opm/input/eclipse/EclipseState/Util/OrderedMap.hpp

```diff
@@ -103,7 +156,16 @@ class OrderedMap {
     T& get(const K& key) {
         auto iter = m_map.find( key );
         if (iter == m_map.end())
-            throw std::invalid_argument("Key not found:");
+        {
+            auto startsWithSame = OrderedMapDetail::findSimilarStrings(key, m_vector);
+            if (!startsWithSame.empty())
+            {
+                startsWithSame = std::string(" Similar entries are ") +
+                    startsWithSame + std::string(".");
```

Comment:
You have libfmt so this is better written as
```C++
startsWithSame = fmt::format(" Similar entries are {}.", startsWithSame);
```

Reply:
Nope, not in installed headers. Using string literals, instead.